tripzone/src/containers/header/Header.js

```diff
@@ -0,0 +1,79 @@
+import React, { useContext, useEffect, useState } from 'react';
+import logo from "../../assets/logo.png"
+import styles from "./Header.module.scss"
+import LoginContext from '../../context/loginContext';
+import Box from '../../components/box/Box';
+import useTimer from '../../hooks/useTimer';
+import { convertISDToEST, getDefaultTime } from '../../utils/getTime';
+
+const Header = () => {
+    console.log("header")
```

Comment:
if these logs are for debugging, it could have been like, 
for components - "Component - component name",
for containers - "Container - container name",
for pages - "Page - page name"